examples/deployments/uc_functions/README.md

```diff
@@ -0,0 +1,27 @@
+# Unity Catalog Integration
+
+## Pre-requisites
+
+```bash
+pip install pandas openai databricks-sdk
+```
+
+## Running the example
+
+First, run the deployments server:
+
+```bash
+# Required to authenticate with Databricks
+export DATABRICKS_HOST="..."   # e.g. https://my.databricks.com
+export DATABRICKS_TOKEN="..."
```

Comment:
Any docs we can link to for how to set these fields?

Reply:
Linked https://docs.databricks.com/en/dev-tools/auth/index.html#supported-authentication-types-by-databricks-tool-or-sdk